### DISCUSSION TITLE
Provide a 64 bit build download

### DISCUSSION THREAD

**Discussion by ctigrisht** [Redot-Engine/redot-proposals#48] [General]
Hey, so I'd need a 64 bit coordinates version of the engine to make a space game, I know godot supported 64 bit but you had to compile it from scratch. I'd like a 64 bit build ready to download.

**Comment by mg0x7BE** [comment]


**Reply by ctigrisht** [reply]
I'm talking about a 64 bit coordinate system, not the process compiled code

**Comment by Arctis-Fireblight** [comment]
I would probably use a chunking system and stick with 32 bit coordinates for that sort of thing.
Godot in general doesnt do well with very large scenes, and if your scene is so large that it cant be represented with 32 bit floats, then it is probably large enough to cause major performance issues and you should consider breaking it into smaller chunks.

You might be able to get away with not doing this in a space game if the world is sparsely populated.

With that being said, we have no plans to ship pre-compiled binaries using double precision vectors. If you absolutely need this functionality you will need to compile the engine yourself with the double precision flag enabled.
